patient-appointment-api.yml

```diff
@@ -97,6 +97,55 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/ProcessingResponse"
+        "403":
+          description: Missing permission
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProcessingResponse"
+  /v1/medication-history/patient-panel:
+    post:
+      tags:
+        - Medication History
+      summary: Patient Panel Load
+      description: "\n# Data Requirements\n\n**Populate all known data**. It is required\
+        \ for processing that as much information as possible is supplied.\n \n##\
+        \ Overview\nSubmits a panel of patients which will be submitted for processing\
```

Comment:
-> Submits a panel of patients for batch processing of medical history information.

Reply:
or 
Submits a panel of patients for medical history information batch processing.

Reply:
I realize this is based on the swagger of another service, sorry I didn't comment there :(